### DISCUSSION TITLE
Early create of pending version without merging earlier one (stabilityDays)

### DISCUSSION THREAD

**Discussion by wyardley** [renovatebot/renovate#21634] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

<img width="832" alt="image" src="https://user-images.githubusercontent.com/10714426/233450950-3f00d102-34c4-4499-8444-ad946e3e7c49.png">

When there's both an existing version and a pending version (in this case, for a Terraform provider) that's more recent than `stabilityDays`, I can see the pending version in the PR, but can't see a way to create an update for the newer version.

We are using the default (`strict`) setting for `internalchecksfilter`.

### What you tried so far.

I know it's hard to come up with a perfect / optimal setup for this.

I looked in the dashboard, and while there are sections for "pending status checks" and "awaiting schedule", I don't see a way to create a PR to update to the pending version sooner.

At one point, I thought maybe it had also been possible to force creation of the updated version, but not 100% sure on that.

In this case, `stabilityDays` isn't intended so much around the possibility of a release being unpublished, but just giving a bit of time for new releases to come out. However, in a case where there's a release and then a quick subsequent bugfix, this can actually result in the opposite issue.

https://docs.renovatebot.com/configuration-options/#internalchecksfilter _seems_ to imply that I should see the suppressed version in the dash, but currently I don't. I may just disable `stabilityDays` for some or all managers, but thought I'd post here to see if this is the expected behavior and / or if there were any other suggestions.

**Comment by rarkins** [comment, marked as answer]
I created #21719 for this feature
